tailwind.config.js

```diff
@@ -1,24 +1,28 @@
+const colors = require('tailwindcss/colors')
+
 module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        'sans': 'Proxima Nova, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
+        sans:
+          'Proxima Nova, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
       },
-      screens: {
-        'print': {'raw': 'print'},
+      screens: { print: { raw: 'print' } },
+      colors: {
+        gray: colors.blueGray,
+        orange: colors.orange,
+        yellow: colors.yellow,
+        green: colors.green,
+        teal: colors.teal,
+        blue: colors.lightBlue,
+        purple: colors.purple,
       },
-    }
-  },
-
-  variants: {
-    width: ['responsive', 'focus'],
-    borderColor: ['responsive', 'hover', 'focus', 'group-hover'],
+    },
   },
-
+  variants: { width: ['responsive', 'focus'], borderColor: ['responsive', 'hover', 'focus', 'group-hover'] },
```

Comment:
I wonder if we can tweak prettier to multi-line this

Reply:
I guess if we did, it would probably also multi-line the arrays so either way it's going to be different than the original.

Personally I'd still prefer the multi-line though.